site-src/v1alpha2/guides/multiple-ns.md

```diff
@@ -63,90 +64,71 @@ administration to ensure that Routes are not over-exposed.
+Gateway so that each HTTPRoute does not also have to configure hostname matching,
+since they are all using the same domain. This also allows these HTTPRoute
+manifests to be reused across production and dev environments where the dev
 environment might be hosted at `foo.dev.corp.example.com`.
-- The Gateway is configured for HTTPS and references the `foo-example-com` Secret. 
-This allows the certificate to be managed centrally for all applications which 
+- The Gateway is configured for HTTPS and references the `foo-example-com` Secret.
+This allows the certificate to be managed centrally for all applications which
 are using this Gateway.
-- It allows any Route in the cluster to use this Gateway because `namespaces.from = All`. 
-This is a permissive method of Route selection since the Routes are given 
-full control to select this Gateway. There are more restrictive forms of Route 
-selection that allow selection on a per-Namespace basis, detailed 
+- It allows any Route in the cluster to use this Gateway because `namespaces.from = All`.
+This is a permissive method of Route selection since the Routes are given
+full control to select this Gateway. There are more restrictive forms of Route
+selection that allow selection on a per-Namespace basis, detailed
 in [Route binding](/concepts/api-overview/#route-binding). The following block
-specifies how this Gateway allows HTTPRoutes from all Namespaces in the 
+specifies how this Gateway allows HTTPRoutes from all Namespaces in the
 cluster to bind to it:
 
 ```yaml
     routes:
-      kind: HTTPRoute
       namespaces:
         from: "All"
+      kinds:
+      - HTTPRoute
 ```
 
-Meanwhile, the store team deploys their route for the `store` Service in the 
+Meanwhile, the store team deploys their route for the `store` Service in the
 `store-ns` Namespace:
 
 ```yaml
-{% include 'v1alpha1/cross-namespace-routing/store-route.yaml' %}
-```
-
-This Route has straightforward routing logic as it just matches for 
-`/store` traffic which it sends to the `store` Service. The following snippet 
-of the [`gateways` field](/v1alpha2/references/spec/#networking.x-k8s.io/v1alpha1.RouteGateways) 
-controls which Gateways this Route can bind to:
-
-```yaml
-  gateways:
-    allow: FromList
-    gatewayRefs:
-    - name: shared-gateway
-      namespace: infra
+{% include 'v1alpha2/cross-namespace-routing/store-route.yaml' %}
 ```
 
-`gateways.allow` can be configured for Gateways in the same Namespace as the
-Route (the default), all Gateways, or a list of specific Gateways. In this
-example the store and site teams decide to reference a specific Gateway. This is
-the least permissive choice which ensures that other Gateways in the cluster
-(perhaps created in the future at some point) will not bind with these Routes.
-If cluster administrators have full control over how Gateways are deployed in a
-cluster then a more permissive binding option could be configured on Routes. The
```

Comment:
@robscott Are we using "Route Binding" or "Route attachment" from a terminology perspective? If latter, please change within the doc for consistency.

Reply:
I think we've historically used the terms interchangeably, but especially with the newer v1alpha2 approach, I personally prefer "attachment"

Reply:
It looks like we're getting a lot of this mixed around, for instance the `site-src/v1alpha2/concepts/api-overview.md` recently changed the nomenclature of the section previously known as `Route Binding` to `Attaching Routes to Gateways` but then inside of that section continues to use the word binding (interchangeably, as rob suggested).

For the purposes of updating this specific guide, I've removed the binding language and just use "attachment" for consistency with the `Attaching Routes to Gateways` section in the concepts docs:

48b31b72f7606564866d0b0af156545469c2f566

Let me know what you think.

Reply:
this guide now uses "route attachment"